content/_index.md

```diff
@@ -1,6 +1,15 @@
 ---
-title: "Open Service Mesh"
-description: "Open Service Mesh documentation and resources."
+title: "Open Service Mesh Documentation"
+description: ""
 type: docs
 
----
\ No newline at end of file
+---
+
+A simple, complete, and standalone service mesh.
+
+Open Service Mesh (OSM) has an architecture based on the [Envoy](https://www.envoyproxy.io/) reverse-proxy sidecar and works by injecting an Envoy proxy as a sidecar container with every application on your Kubernetes cluster. OSM also includes a fully featured control plane and can be configured via SMI APIs.
```

Comment:
I wonder if instead of "based on" we want to incorporate language from https://github.com/openservicemesh/osm-www/blob/main/themes/clean-landing/layouts/partials/points.html#L6 more like this:

`OSM runs on <a href="https://kubernetes.io/">Kubernetes</a>. The control plane implements <a href="https://www.envoyproxy.io/">Envoy's</a> xDS and is configured with <a href="https://smi-spec.io/">SMI</a> APIs. OSM injects an Envoy proxy as a sidecar container next to each instance of an application.</p>`

Reply:
agreed, fixing